src/fidgets/farcaster/components/Embeds/WarpcastImage.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable @next/next/no-img-element */
+import React, { useState } from "react";
+import { PhotoIcon } from "@heroicons/react/24/solid";
+import { Skeleton } from "@/common/components/atoms/skeleton";
+import { Label } from "@/common/components/atoms/label";
+
+const getImageViaCdnUrl = (imgUrl: string, skipCdn: boolean) => {
+  if (imgUrl.startsWith("https://imagedelivery.net")) return imgUrl;
+
+  if (!skipCdn && imgUrl.includes("imgur.com")) {
+    const fileSuffix = imgUrl.split(".").slice(-1)[0];
+    return `https://res.cloudinary.com/merkle-manufactory/image/fetch/c_fill,f_${fileSuffix}/${imgUrl}`;
+  }
+  return imgUrl;
+};
+
+const WarpcastImage = ({ url }: { url: string }) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [skipCdn, setSkipCdn] = useState(false);
+
+  const onImageLoad = (e: any) => {
+    setIsLoading(false);
+    e.currentTarget.style.display = "block";
+  };
+
+  const renderLoadingPlaceholder = () => {
+    return (
+      <Skeleton className="h-36 w-48 object-left relative block rounded-lg py-10 text-center">
+        <PhotoIcon className="mx-auto h-12 w-12 text-foreground/70" />
+        <Label>Loading image...</Label>
+      </Skeleton>
+    );
+  };
+
+  return (
+    <>
+      <img
+        className="mt-2 max-h-48 md:max-h-72 object-left rounded-md"
+        style={{ display: "none" }}
```

Comment:
No display?

Reply:
Not sure, again a herocast copy thing

Reply:
Looking at the code, it is weird hack that the original author of this file did to not render the image until it is loaded. `onImageLoad` (defined further up in the file) changes the style of this element to block. I have updated this to make the display style just conditional on the `isLoading` state like it should've been done